spec/factories/users.rb

```diff
@@ -69,7 +69,6 @@
     device_updated_at { Time.current }
     sync_approval_status { User.sync_approval_statuses[:denied] }
     email_authentications { build_list(:email_authentication, 1, email: email, password: password) }
-    organization
```

Comment:
Ah, good catch, thanks for this.

Reply:
Ooh neat, and associated specs too. I'll merge `master`, thanks!